README.md

```diff
@@ -39,89 +39,3 @@ development). PoET is only crash fault tolerant, so if any nodes in the
 network exhibit [Byzantine
 behaviour](https://en.wikipedia.org/wiki/Byzantine_fault_tolerance#Byzantine_Generals'_Problem),
 it causes issues with consensus.
-
-
-## Features
-The following features have been implemented:
-
-+ [x] **Normal case operation:** Handling transactions when the network is functioning normally
-+ [x] **View changes:** When a primary node is considered faulty (crashed or
-  malicious), the network changes views and a new primary is elected.
-+ [x] **Log garbage collection:** Every so often, message logs should be garbage
-  collected so as to not take up too much space.
-+ [x] **Testing improvements:** Presently, a liveness test up to 55 blocks has
-  been performed on a network of four nodes. Unit tests are also included for
-  each individual component of the algorithm.
-
-The following features are desired (not a comprehensive list):
-+ [ ] **Allow network changes:** Right now, the network is assumed to be
-  static. Peers are introduced through on-chain settings, and the peer list
-  does not change during network operation.
-+ [ ] **Persistent storage:** Nodes should be able to recover from crashes by
-  saving their logs in persistent storage instead of keeping everything in
-  memory
-+ [ ] **Documentation:** Use existing Sawtooth doc generation to create
-  documentation for this project
-
-## Running PBFT
```

Comment:
Could we keep a section on running PBFT? Even if it's just something that says "look at these scripts in `tests/`" or "here's a link to the hosted documentation that says how to run it", it would be nice to provide new users some guidance in the README.

Reply:
We'll link to the documentation from the README, but the documentation isn't hosted yet. When it is, we'll include the link here.